README.md

```diff
@@ -1,7 +1,5 @@
 # ArcGIS Runtime Java SDK Samples
 
- [ ![Download](https://api.bintray.com/packages/esri/arcgis/arcgis-java/images/download.svg) ](https://bintray.com/esri/arcgis/arcgis-java/_latestVersion) ![Gradle build](https://github.com/Esri/arcgis-runtime-samples-java/workflows/Java%20CI%20with%20Gradle/badge.svg)
```

Comment:
I don't think we should be removing these badges, we can update them. The second badge is just a status tracker for our Github CI checks. The first badge points to where to download the latest release. We could try just replacing the first url link from:

https://bintray.com/esri/arcgis/arcgis-java/_latestVersion

to

https://esri.jfrog.io/artifactory/arcgis